tests/remote_attestation/maa/sgxlkl_enclave/oe_private/cert.c

```diff
@@ -0,0 +1,901 @@
+#include <mbedtls/ctr_drbg.h>
```

Comment:
As discussed, the functionalities of this file should ideally be exposed through OE host verify.